schemas/biomes_schema.json

```diff
@@ -12,6 +12,10 @@
         "type": "integer",
         "minimum": 0
       },
+      "textId": {
```

Comment:
That's the same as name isn't it ?

Reply:
It probably is, will remove it if so.

Reply:
For biomes, the name and textId are not the same, name is actually the display name of the biome (Ex: Frozen Ocean) and the textId is well, the id (ex: frozen_ocean)

Reply:
Ok. Could you call textid name and what is currently name would become "displayName" ?
That won't break anything since we don't use biomes that much.

Reply:
Yeah expect that changing the schema to add displayName will basically make all the biomes.json prior to 1.13 fail

Reply:
Just make it optional

Reply:
Fixed!